src/main/java/racing/util/InputValidator.java

```diff
@@ -1,5 +1,62 @@
 package racing.util;
 
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
 public class InputValidator {
+    private final static String NUMERIC = "^[1-9]+$";
+    private final static String REG = "^[`~!@#$%^&*()-_=+]";
+
+// 자동차 이름은 5자를 초과할 수 없다.
+// 자동차 이름은 쉼표(,)를 기준으로 구분한다.
+    public String[] validateInputCarNames(String inputCarNames) {
+        String carNames = StringUtils.deleteWhitespace(inputCarNames);
+
+        String[] carNamesArray = makeCarNamesArray(carNames);
+
+//        isContainSpecialChar(carNamesArray);
+        isCarAtLeatTwo(carNamesArray);
+        isDuplicateCarNames(carNamesArray);
+
+        return carNamesArray;
+    }
+
+    public void validateInputPlayTimes(String inputPlayTime) {
+        if(!inputPlayTime.matches(NUMERIC)) {
+            throw new IllegalArgumentException("숫자만 입력할 수 있습니다.");
+        }
+    }
+
+    public String[] makeCarNamesArray(String carNames) {
+        String[] carNamesArray = carNames.split(",");
+
+        return carNamesArray;
+    }
+
+//    public void isContainSpecialChar(String[] carNamesArray) {
+//        for(String carName : carNamesArray) {
+//            if(!carName.matches(REG)) {
+//                throw new IllegalArgumentException("특수문자는 입력할 수 없습니다.");
+//            }
+//        }
+//    }
+
+    public void isCarAtLeatTwo(String[] carNamesArray) {
```

Comment:
is~로 시작하는 함수는 보통 리턴형을 boolean으로 하는것이 관행입니다. 여기서는 validation check를 하기 때문에 명명을 조금 다르게 하는게 좋을것 같아요
```suggestion
    public void validateIfCarsAtLeastTwo(String[] carNamesArray) {
```
```suggestion
    public void throwIfCarsLessThanTwo(String[] carNamesArray) {
```